nilearn/datasets/atlas.py

```diff
@@ -13,7 +13,7 @@
 from nibabel import freesurfer, load
 from sklearn.utils import Bunch
 
-from .._utils import check_niimg, fill_doc, logger
+from .._utils import check_niimg, fill_doc, logger, rename_parameters
```

Comment:
```suggestion
from nilearn._utils import check_niimg, fill_doc, logger, rename_parameters
```

Reply:
maybe this is just me but I find relative import confusing, so taking the opportunity to fix this one